src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -0,0 +1,42 @@
+package com.crud.tasks.controller;
+
+import com.crud.tasks.domain.TaskDto;
+import lombok.extern.java.Log;
+import org.springframework.web.bind.annotation.*;
+import java.util.*;
+
+@Log
+@RestController
+@RequestMapping("/v1/task")
+public class TaskController {
```

Comment:
Please read this article https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9
Especially second point, and refactor your code. I know that you used example from course, but example presented there is anti-pattern.